tests/console/consoletest_setup.pm

```diff
@@ -71,6 +71,10 @@ sub run {
 
     # init
     check_console_font if has_ttys();
```

Comment:
my $res = script_run 'echo ${SHELLOPTS} | grep -w pipefail';
    script_run '. /etc/bash.bashrc.local' if $res;

Reply:
I guess you want to source that rc file if the command fails (return code !=0).

Reply:
Isn't that what it already does?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2217